Tasks/merge.js

```diff
@@ -1,16 +1,13 @@
 // Merge two dictionaries
 
-merge_two_objects = function merge_two_objects(object_1, object_2) {
-  object_3 = [{}][0];
-  for (attribute_name in object_1) {
-    object_3[attribute_name] == object_1[attribute_name];
-    object_3[attribute_name] = object_1[attribute_name];
+const merge = (object1 = {}, object2 = {}) => {
+ const object3 = { ...object1};
```

Comment:
It's not critical, but I suggest to name the array we return something more understandable:
```suggestion
 const result = { ...object1};
```